clang/test/Driver/sycl-offload.c

```diff
@@ -418,13 +418,16 @@
 /// Ahead of Time compilation for gen
 // RUN:   %clang -target x86_64-unknown-linux-gnu -ccc-print-phases -fsycl -fsycl-targets=spir64_gen-unknown-linux-sycldevice %s 2>&1 \
 // RUN:    | FileCheck %s -check-prefixes=CHK-PHASES-AOT,CHK-PHASES-GEN
+// RUN:   %clang -target x86_64-unknown-linux-gnu -ccc-print-phases -fsycl -fsycl-targets=spir64_x86_64-unknown-linux-sycldevice %s 2>&1 \
```

Comment:
Comment is missed that this runs AOT for CPU (or otherwise comment at 418 is not quite accurate)

Reply:
Mike, I think you also remove the comment at 418 to make the comments consistent.